### DISCUSSION TITLE
Brightness adjustment and keyboard backlit on/off

### DISCUSSION THREAD

**Discussion by cpot** [AdnanHodzic/auto-cpufreq#757] [Ideas]
If i understood well auto-cpufreq replaces and improve gnome power management services and is aimed to extend battery autonomy and auto-cpufreq does it well. I use it on Zorin (Ubuntu kind).
However it seems that it does not reduce brightness while on battery which could extend even more the autonomy.
Also, keyboard backlit could be perhaps governed.

Thanks for any thoughts

**Comment by AdnanHodzic** [comment]
Hm, this would indeed be an interesting feature to have. I would love if i.e screen brightness automatically dimmed based on time of the day and/or other criteria (same with keyboard backlit). I suggest you [open an issue and prepend  "[Feature request]"](https://github.com/AdnanHodzic/auto-cpufreq/issues) to it and explain what you would like.

In meantime, there's a [ GNOME Extension "Dim on Battery Power"](https://extensions.gnome.org/extension/947/dim-on-battery-power/) which I used for this purpose. But unfortunately it hasn't been updated to support latest version of GNOME.

Regarding keyboard backlight, I keep it off, and on X1 Carbon it's one shortcut away from being turned on "fn + space".

**Reply by cpot** [reply]
Thank you, i will open the feature request !

> In meantime, there's a GNOME Extension "Dim on Battery Power" which I used for this purpose. But unfortunately it hasn't been updated to support latest version of GNOME.

Incredible that it has not been included in the GNOME power management service (however i disabled it as you recommended that is why it would be great in auto-cpufreq )

PS: forgot the mention that i am using a LENOVO T14 G3 AMD
